manifests/charts/istio-control/istio-discovery/templates/role.yaml

```diff
@@ -24,3 +24,7 @@ rules:
   resources: ["configmaps"]
   verbs: ["delete"]
 
+# For gateway deployment controller
```

Comment:
I assume this has been around for enough time ? 

Does it make a huge difference - and is it as easy to debug as the old one ?